composer.json

```diff
@@ -14,14 +14,14 @@
         }
     ],
     "require": {
-        "php": "^5.5 || ^7.0",
+        "php": "^5.6 || ^7.0",
```

Comment:
hmm thats weird (: lets include also @danrot and @chirimoya in this discussion. maybe we can include this not before 2.0.